submissions/BlessingOkoro/html/AboutUs.html

```diff
@@ -4,30 +4,77 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="css/style.css" type="text/css" rel="stylesheet">
```

Comment:
The CSS files and the images are not loaded on the page. The path to those files appear to be wrong. Please check again